pythonx/jedi_vim.py

```diff
@@ -1027,11 +1030,15 @@ def do_rename(replace, orig=None):
         vim.current.buffer[r.line - 1] = (r_line[:r.column] + replace +
                                           r_line[r.column + len(orig):])
 
+    vim_command('w')
```

Comment:
I don't think we should do this. We should not save randomly. This is the user's responsibility.

Reply:
That makes sense, but for creating new buffers the function `new_buffer` saves anyways for all buffers except the final one, and the buffer needs to be saved in order to return to the original buffer where the refactor happened.

Reply:
I should disable this behavior for tabs though

Reply:
Saving should at least get skipped with `set hidden` then maybe (a sane default).